### DISCUSSION TITLE
How to delete entries from being shown in the docs directory

### DISCUSSION THREAD

**Discussion by neonone1234** [thuliteio/doks#1021] [Q&A]
Hi, everyone!
I am sorry in advance if this was previously asked, I did my best to find an answer and I couldn't. 
I created a category in docs, and costumized the index of the category to show a table, where I link all the entries in the docs the way I want. I only need to remove the automatic list of entries that show up on the index page. But I still want the automatic list of entries to be shown in the sidebar when you are inside the entry. I will provide screenshots for better explanation.

![Screen Shot 2023-03-04 at 23 38 57](https://user-images.githubusercontent.com/99328716/222927775-465ec915-e16d-4c25-b803-901d8352d07d.png)

![Screen Shot 2023-03-04 at 23 40 03](https://user-images.githubusercontent.com/99328716/222927758-76a56f13-5ec0-44cf-879b-ac2e8e421573.png)

**Comment by pulkitkrishna00** [comment]
How did you add the table to that page? Did you modify layouts/docs/list.html? You can change that file to customize the look of those pages, including removing the default things. Remove the following lines to remove the listing:
```html
<div class="card-list">
	{{ $currentSection := .CurrentSection }}
	{{ range where .Site.RegularPages.ByTitle "Section" .Section }}
		{{ if in (.RelPermalink | string) $currentSection.RelPermalink }}
			<div class="card my-3">
				<div class="card-body">
					<a class="stretched-link" href="{{ .RelPermalink }}">{{ .Params.title | title }} &rarr;</a>
				</div>
			</div>
		{{ end }}
	{{ end }}
</div>
```
But, it will modify it for all the category pages.

**Reply by h-enk** [reply]
See also https://github.com/h-enk/doks/discussions/1024#discussioncomment-5238976
